Tests/GRPCProtobufCodeGenTests/ProtobufCodeGenParserTests.swift

```diff
@@ -199,4 +216,15 @@ extension Google_Protobuf_FileDescriptorProto {
       $0.syntax = "proto3"
     }
   }
+
+  static func makeEmpty(_ name: String) -> Google_Protobuf_FileDescriptorProto {
```

Comment:
It's not really empty, it has a name and package so this it's a bit misleading. An `init` with `name` and `package` as parameters would make this a bit clearer.